cafebabel/core/views.py

```diff
@@ -8,6 +8,6 @@ def home():
     return render_template('home.html')
 
 
-@cores.route('/uploads/<filename>')
+@cores.route('/uploads/<path:filename>')
```

Comment:
Mhhh… Now that I see this, shouldn't it be manage by the Nginx?

Reply:
Sure, nginx or Apache must take over that path prior it hits Flask in production/preprod.